### ISSUE TITLE
Grid search on Prophet is on test set itself

### ISSUE THREAD

**Issue by swhustla** [swhustla/comparison_of_time_series_methods#36]
Ideally it should be on a validation set, part of the training set; and then assessed on the test set
